content/docs/ops/maintenance-list.md

```diff
@@ -16,6 +16,18 @@ visibility with the people who might best help.
 
 - Join/unmute [`#cg-support`](https://gsa-tts.slack.com/messages/cg-support/)
 
+## Support Rotations
+
+The cloud.gov team utilizes a staggered pair rotation for support. Support assignments last two weeks. One team member exits support duty each week, with a new member starting their two-week rotation. With the other support team member in the middle of their two-week rotation, we ensure continuity of situational awareness, task handoffs, and work in progress.
+
+The support team is not required to "pair" in the traditional sense unless the task as hand necessitates doing so. Having a pair rather than a single person on support allows us to better serve our customers by:
+
+- improving  situational awareness and communication through necessary, regular check-ins
```

Comment:
I would capitalize these for grammar's sake.

Reply:
done